### PR TITLE
Update UmbracoServices to v9

### PR DESCRIPTION
**Pull request by elit0451**
## Details

- index-v9.md 
  - Changed code snippets and wording 
  - Some of the links at the bottom of index-v9.md are not changed as the articles have not been migrated + Umbraco TV videos are about v7

</br>

Articles under services available:
* AuditService
  - removed **Services property** as it uses `Services` property which we no longer use
  - removed **Static accessor** section as it was using `Current` which we don't have any more

* ConsentService

* ContentService

* ContentTypeService

* ApplicationTreeService
  - removed - was dealing with tree registrations in the ~/Config/trees.config file (not working with such file anymore)

* DataTypeService
  - removed **Services property** as it uses `Services` property which we no longer use
  - removed **Static accessor** section as it was using `Current` which we don't have any more

* EntityService
  - removed **Services property** as it uses `Services` property which we no longer use
  - removed **Static accessor** section as it was using `Current` which we don't have any more

* FileService
  - removed **Services property** as it uses `Services` property which we no longer use
  - removed **Static accessor** section as it was using `Current` which we don't have any more

* LocalizationService
  - removed **Services property** as it uses `Services` property which we no longer use
  - removed **Static accessor** section as it was using `Current` which we don't have any more
 
*  MacroService
   - removed **Services property** as it uses `Services` property which we no longer use
   - removed **Static accessor** section as it was using `Current` which we don't have any more

* MediaService
  - removed **Services property** as it uses `Services` property which we no longer use
  - removed **Static accessor** section as it was using `Current` which we don't have anymore
  - check code snippet from **Creating a new media from an uploaded file** and note ❗ ❗  (the only thing that needs to be updated in order to completely mark the article as updated)

* MemberService
 
* MemberTypeService

* MemberGroupService

### CODE REVIEW

**Review by bergmania — CHANGES_REQUESTED** [review]
Here is my suggested changes.. What do you think @jonat123

File: Fundamentals/Code/Umbraco-Services/index-v9.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+### More information
+- [Umbraco Services API reference](../../../Reference/Management/Services/)
+- [Umbraco Events reference](../../../Reference/Events/) <!--- (this should change) -->
+- [Routes and controllers](../../../Reference/Routing/)
+
+### Umbraco TV
+- [Chapter: Content API](https://umbraco.tv/videos/umbraco-v7/developer/fundamentals/content-api/)
+- [Chapter: Media API](https://umbraco.tv/videos/umbraco-v7/developer/fundamentals/media-api/)
+- [Chapter: Member API](https://umbraco.tv/videos/umbraco-v7/developer/fundamentals/member-api/)
```
Comment (jonat123) [inline Fundamentals/Code/Umbraco-Services/index-v9.md:266]:
```suggestion

```

Comment (jonat123) [reply Fundamentals/Code/Umbraco-Services/index-v9.md:266]: Lets just remove the videos for V7, as i don't think it makes sense that we have videos for V7 in a article for V9, plus we are going to move away from Umbraco TV anyways so the links wont work anyways

**Review by jonat123 — COMMENTED** [review]
@elit0451 Looks good, I made a suggestion that we remove the V7 videos as it makes no sense to have v7 videos in a v9 article, furthermore, we are closing down Umbraco TV and moving to youtube so the links won't work in the future anyway.

@bergmania I like your suggestions so I have committed them to the PR :)

**Review by bergmania — APPROVED** [review]

**Comment by bergmania** [comment]
All good.. Elitsa is on vacation, so I approved your suggestion.. IMO this is ready to merge.

**Comment by jonat123** [comment]
Great! I'll get it merged in 🙂

**Pull request merged**